.github/workflows/publish-docker-image.yml

```diff
@@ -59,7 +64,7 @@ jobs:
           # See https://github.com/docker/metadata-action#tags-input for the
           # format this should take.
           # DUE TO JSON LIMITATIONS, NEWLINES MUST BE EXPLICITLY ADDED AS \n
-          additional_image_tags: ''
+          additional_image_tags: 'type=raw,value=${{ inputs.docker_tag }}'
```

Comment:
**non-blocking,question**: Docker image names come with "repo" and "tag" parts, but sometimes the nomenclature is ambiguous. I can see `target_docker_repository` is specified separately above. Just to make sure, what's gonna be the end "full image name" when I run this workflow with `docker_tag` set to `foo-123`?
1. `hub.stage.nexus.orquestra.io/zapatacomputing/orquestra-sdk-base:foo-123`
2. `foo-123`

Reply:
It will be the first one. Added more clarification to the parameter description.